### DISCUSSION TITLE
Добавить git blame ignore

### DISCUSSION THREAD

**Discussion by edbond88** [doka-guide/platform#868] [Идеи]
### Проблема
После того как будет смержен https://github.com/doka-guide/platform/pull/829, то для тех кто юзает метод `git blame` будет  не очень удобно им пользоваться, т.к. большая часть js файлов будет с коммитом, который выполнял eslint\prettier fix.

### Решение
Можно создать `.git-blame-ignore-revs` в который записываются айди коммитов которые нужно игнорировать. 
После этого прописывается`git config blame.ignoreRevsFile .git-blame-ignore-revs` в настройках гита для проекта. 
И теперь при использовании `git blame` будут игнорироваться коммиты из файла.

Редакторы кода, кстати тоже автоматически подтягивают ваш гитконфиг и не отображают коммиты из файла.

Вот тут более подробно описана проблема и способы ее решения: [Дружим git и prettier](https://www.youtube.com/watch?v=CbYQkuh5ww0)

### Вопросы
1. Выполнять ли команду `git config blame.ignoreRevsFile .git-blame-ignore-revs` в postinstall хуке?
2. Желательно об этом описать в документации по настроке проекта.

ps: уже опробовал этот подход у себя на проекте - очень понравилось.

**Comment by monochromer** [comment]
> Можно создать `.git-blame-ignore-revs` в который записываются айди коммитов которые нужно игнорировать.

Id коммитов будут записываться автоматически?

**Reply by edbond88** [reply]
Нет, это будет делаться руками. Таких комитетов скорей всего будет пару штук, и они будут добавляться очень редко. Можно даже оставлять в файле краткий комментарий под каждым

**Reply by monochromer** [reply]
В таком случае, такие коммиты нужно выносить в отдельные pull request'ы.

**Reply by edbond88** [reply]
Не обязательно в отдельный пул реквесты. Они могут быть в составе одного пул реквеста. 

Вот как тут я делал:
![image](https://user-images.githubusercontent.com/1949927/149902038-0c23e43d-848d-4de7-93f7-87bc228316ca.png)

Хотя такие операции по глобальному линтигу будут скорее всего редкостью, то я бы отдельные ПРы делал.

**Reply by monochromer** [reply]
По умолчанию в репозитории стоит методика слияния "Squash", поэтому все коммиты будут объединены в один. Нужно как-то напоминать об этом.

**Reply by edbond88** [reply]
Получается, что нужно отдельные ПРы делать
